mmdet3d/core/visualizer/show_result.py

```diff
@@ -98,7 +99,23 @@ def show_result(points,
 
         vis = Visualizer(points)
         if pred_bboxes is not None:
-            vis.add_bboxes(bbox3d=pred_bboxes)
+            if pred_labels is None:
+                vis.add_bboxes(bbox3d=pred_bboxes)
+            else:
+                palette = np.random.randint(
+                    0, 255, size=(pred_labels.max() + 1, 3)) / 256
+                S = {}
+                for j in range(len(pred_labels)):
+                    i = int(pred_labels[j].numpy())
+                    if S.get(i) is None:
+                        S[i] = []
+                    S[i].append(pred_bboxes[j])
+                for i in S:
+                    vis.add_bboxes(
+                        bbox3d=np.array(S[i]),
```

Comment:
A little bit confused with S. If it is from label, why it is passed to bbox3d?

Reply:
S is a dictionary, which key is lable and value is list of bounding boxes corresponding to label.
Since I've revised a scripts, please check again.